optimize/backend/src/main/java/io/camunda/optimize/service/db/repository/os/ImportRepositoryOS.java

```diff
@@ -125,7 +125,7 @@ private BulkOperation addPositionBasedImportIndexRequest(
       final PositionBasedImportIndexDto optimizeDto) {
     LOG.debug(
         "Writing position based import index of type [{}] with position [{}] to opensearch",
-        optimizeDto.getEsTypeIndexRefersTo(),
+        optimizeDto.getDbTypeIndexRefersTo(),
         optimizeDto.getPositionOfLastEntity());
     // leaving the prefix "es" although it is valid for ES and OS,
```

Comment:
I believe this comment can be removed now

Reply:
yep, thanks for spotting this!